org.kde.ark.json

```diff
@@ -105,6 +105,94 @@
                 }
             ]
         },
+        {
+         "name": "gnustep-make",
+         "buildsystem": "simple",
+         "build-commands": [
+            "./configure --prefix=/app",
+            "make -j $FLATPAK_BUILDER_N_JOBS install"
+         ],
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://github.com/gnustep/tools-make/archive/make-2_8_0.tar.gz",
+               "sha256": "a9df9b0d756a93306a8e0cb9e127f5688a459dad3d356cd760565833da605825"
+            }
+         ]
+        },
+        {
+         "name": "gnustep",
+         "buildsystem": "simple",
+         "build-commands": [
+            "./configure --prefix=/app",
+            "make -j $FLATPAK_BUILDER_N_JOBS install",
+            "ls /app/include"
+         ],
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://github.com/gnustep/libs-base/releases/download/base-1_27_0/gnustep-base-1.27.0.tar.gz",
+               "sha256": "8803fa1fdf23f90264a81ad2d9fbe97ba66d6e313e396a28fd7c10d2caceb283"
+            }
+         ]
+        },
+        {
+         "name": "unarchiver",
+         "buildsystem": "simple",
+         "build-commands": [
+            "sed -s -i 's#/usr/include/GNUstep#/app/include#g' */Makefile.linux && cd XADMaster && make -j $FLATPAK_BUILDER_N_JOBS -f Makefile.linux",
+            "install -m755 XADMaster/unar XADMaster/lsar /app/bin/"
+         ],
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://cdn.theunarchiver.com/downloads/TheUnarchiverSource.zip",
```

Comment:
Arch Linux moved to https://github.com/MacPaw/XADMaster as upstream: https://github.com/archlinux/svntogit-community/blob/packages/unarchiver/trunk/PKGBUILD

Reply:
I'll open an issue for this one.